docs/style.css

```diff
@@ -16,22 +16,122 @@ You should have received a copy of the GNU General Public License
 along with this program. If not, see <https://www.gnu.org/licenses/>.
 */
 
+/* Basic elements */
 html, body {
```

Comment:
"Everything but cgit looking good"

story of my life hahahaha

(no changes requested with this comment)